packages/frameworks/vue/src/avatar/Avatar.vue

```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
+import type { VNode } from 'vue'
+import { ark, type HTMLArkProps } from '../factory'
+import type { Assign } from '../types'
+import { AvatarProvider } from './avatar-context'
+import { props as avProps } from './avatar.props' // Name change was needed to not have double props
```

Comment:
We can edit a vue props script to generate the object named `componentProps` instead of props.

Reply:
I made a change into renaming the props in `avatar.props.ts` to `componentProps`.